modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,320 @@
+        "gcp": {
+            "subnet": {
+                "name":"<byovpc_subnet_name>",
+                "secondary_ipv4_range_pods": { "name": "<byovpc_subnet_pods_range_name>" },
+                "secondary_ipv4_range_services": { "name": "<byovpc_subnet_services_range_name>" },
+                "k8s_master_ipv4_range": "<byovpc_subnet_master_range>"
+            },
+            "psc_nat_subnet_name": "byovpc_psc_nat_subnet_name"
+            "agent_service_account": { "email": "<byovpc_agent_service_acc_email>" },
+            "connector_service_account": { "email": "<byovpc_connectors_service_acc_email>" },
+            "console_service_account": { "email": "<byovpc_console_service_acc_email>L" },
+            "redpanda_cluster_service_account": { "email": "<byovpc_redpanda_service_acc_email>" },
+            "gke_service_account": { "email": "<byovpc_gke_service_acc_email>" },
+            "tiered_storage_bucket": { "name" : "<byovpc_tiered_storage_bucket>" },
+        }
+    }
+}
+EOF`
+
+curl -vv -X POST \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_POST_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters
+----
+
+== Enable PSC on an existing BYOC cluster
+
+. In the Redpanda Cloud UI, go to the cluster overview and copy the cluster ID from the **Details** section.
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Update the Redpanda Cloud Agent IAM role. This step is required only for clusters with xref:./vpc-byo-gcp.adoc[customer-managed resources].
++
+To allow the agent to create and manage the service attachment, add the following permissions to its IAM role:
++
+[,bash]
+----
+compute.forwardingRules.use
+compute.regionOperations.get
+compute.serviceAttachments.create
+compute.serviceAttachments.delete
+compute.serviceAttachments.get
+compute.serviceAttachments.list
+compute.serviceAttachments.update
+compute.subnetworks.use
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to include the newly-created PSC NAT subnet.
++
+Set the following environment variable:
++
+--
+* `PSC_NAT_SUBNET_NAME`: The name of the PSC NAT subnet. Use the fully-qualified name, for example `"projects/<project>/regions/<region>/subnetworks/<subnet-name>"`.
+--
++
+[,bash]
+----
+export ACCEPT_LIST='[]'
+export PSC_NAT_SUBNET_NAME='<psc-nat-subnet-name>'
+export CLUSTER_PATCH_BODY=`cat << EOF
+{
+    "customer_managed_resources": {
+        "gcp": {
+            "psc_nat_subnet_name": "$PSC_NAT_SUBNET_NAME"
+        }
+    }
+}
+EOF`
+curl -v -X PATCH \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to enable PSC.
```

Comment:
Use a standard. Whatever we choose, I would prefer to always use this kind of construct: 'Make a request to x endpoint', 'Call the x endpoint'.